Doppler.ImageAnalyzer.Api/Controllers/DopplerControllerBase.cs

```diff
@@ -25,9 +28,46 @@ protected ActionResult HandleResponse<T>(Response<T> response, string logMessage
     {
         if (!response.IsSuccessStatusCode)
         {
+            LogImageAnalysisResponseNonSuccess((int)response.StatusCode, response.ValidationIssue);
             return StatusCode((int)response.StatusCode, response.ValidationIssue);
         }
 
+        if (response.Payload != null)
+        {
+            var imagesAnalysis = (List<ImageAnalysisResponse>)(object)response.Payload;
+            LogImageAnalysisResponseSuccess((int)response.StatusCode, imagesAnalysis);
+        }
+
         return Ok(response.Payload);
     }
+
+    private void LogImageAnalysisResponseNonSuccess(int statusCode, ResponseErrorDetails errorDetails)
+    {
+        _logger.LogInformation(
+            "Analisys Response => Status Code: {AnalysisResponse_StatusCode} - Title: {AnalysisResponse_ErrorTitle} - Exception Message: {AnalysisResponse_ExceptionMessage}",
+            statusCode,
+            errorDetails.Title,
+            errorDetails.ExceptionMessage
+        );
+    }
+
+    private void LogImageAnalysisResponseSuccess(int statusCode, List<ImageAnalysisResponse> imagesAnalysis)
+    {
+        _logger.LogInformation(
+            "Analisys Response => Status Code: {AnalysisResponse_StatusCode} - Number of Images: {AnalysisResponse_ImagesCount} - Results: {AnalysisResponse_Result}",
+            statusCode,
+            imagesAnalysis.Count,
+            imagesAnalysis.Select((ia, index) => (FormatImageAnalysisResponseLogging(index, ia)))
```

Comment:
Creo que se puede usar un serializador que serilog trae de serie en lugar de hacerlo a mano. @RodrigoPereyraDiaz está más familiarizado con eso.

Reply:
Ok, @RodrigoPereyraDiaz have you any idea how it would be?

Reply:
Done.

Reply:
👍